include/slang/binding/BindContext.h

```diff
@@ -66,7 +66,12 @@ enum class BindFlags {
 
     /// Expression is for a static variable's initializer. References to automatic
     /// variables will be disallowed.
-    StaticInitializer = 1 << 9
+    StaticInitializer = 1 << 9,
+
+    /// Streaming operator is allowed in assignment target, assignment source and bit-stream casting
```

Comment:
According to IEEE, a streaming_concatenation can also be a stream_expression in another streaming_concatenation.
Does this need to be handled?

Reply:
In addition, streaming_concatenation as a stream_expression in another streaming_concatenation is covered in the unit tests.
```c++
        "shortint a; byte b[2]; int c = {<<3{a, {<<5{b}}}};",
        "shortint a; byte b[2]; int c; assign {<<3{{<<5{b}}, a}}=c;",

    {<<16{{<<4{a[0]}},a[1],{<<3{a[2]}}}} = bar;
    {>>{a,{<<{b}},c}} = bar;
```

Reply:
While the code is correct, you pointed out that the comment is incomplete. I polish the comment to be
```c++
   /// Streaming operator is allowed in assignment target, assignment source, bit-stream casting
   /// argument, or stream expressions of another streaming concatenation. This flag is cleared for
   /// nested subexpressions, unless they are directly parenthesized.
```
and it will be available in the next pull request.